content/docs/user-guide/merge-conflicts.md

```diff
@@ -0,0 +1,198 @@
+# Merge Conflicts
+
+Sometimes multiple members of a team might want to edit the same file/directory
+that is tracked by DVC. And when the time comes to merge their changes together
+one will run into a merge conflict.
+
+## dvc.yaml
+
+This one is no different from your regular merge conflicts in code.
+
+```
```

Comment:
````suggestion
```yaml
````

Reply:
Or maybe ` ```diff ` ?

Reply:
That's not diff, so using yaml for now.

Reply:
True.

Reply:
True. Maybe `git`? Here's a full list: https://prismjs.com/#supported-languages

YAML is fine too, probably.